README.md

```diff
@@ -1 +1,63 @@
-# angular-what-is-a-custom-directive-readme
\ No newline at end of file
+# What is a custom Directive?
+
+## Overview
+
+One of the massive powerful aspects of Angular is the ability to actually be able to create our own directives.
+
+## Objectives
+
+- Describe what components are
+- Describe custom Directives
+- Create a custom Directive
+
+## Components
+
+Nowadays, most frontend applications are made up of small, reusable components. If we imagine a car, it would be made up with components such as wheels, windows, engine, etc.
```

Comment:
Great analogy here. Hammer this home by listing some components that a web app is made up of - navbar, sidebar, header, footer, etc.